lib/features/composer/presentation/composer_controller.dart

```diff
@@ -374,22 +374,42 @@ class ComposerController extends BaseController with DragDropFileMixin implement
     });
 
     _subscriptionOnDragEnter = html.window.onDragEnter.listen((event) {
-      mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.active;
+      event.preventDefault();
+
+      if (_validateFilesTransfer(event.dataTransfer.types)) {
+        mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.active;
+      }
     });
 
     _subscriptionOnDragOver = html.window.onDragOver.listen((event) {
-      mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.active;
+      event.preventDefault();
+
+      if (_validateFilesTransfer(event.dataTransfer.types)) {
+        mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.active;
+      }
     });
 
     _subscriptionOnDragLeave = html.window.onDragLeave.listen((event) {
-      mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.inActive;
+      event.preventDefault();
+
+      if (_validateFilesTransfer(event.dataTransfer.types)) {
+        mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.inActive;
+      }
     });
 
     _subscriptionOnDrop = html.window.onDrop.listen((event) {
-      mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.inActive;
+      event.preventDefault();
+
+      if (_validateFilesTransfer(event.dataTransfer.types)) {
+        mailboxDashBoardController.localFileDraggableAppState.value = DraggableAppState.inActive;
+      }
     });
   }
 
+  bool _validateFilesTransfer(List<dynamic>? types) {
+    return types?.any((type) => type == 'Files') ?? false;
```

Comment:
```suggestion
    return types?.every((type) => type == 'Files') ?? false;
```

Reply:
No, we just need the type list to contain at least one type of `Files`, we will give priority to drag & drop files.